kano_peripherals/wrappers/detection.py

```diff
@@ -40,6 +54,9 @@ def is_pi_hat_plugged(with_dbus=True, retry_count=5):
     Returns:
         bool: Whether the PiHat is plugged in or not
     """
+    from kano_peripherals.pi_hat.driver.service import PiHatService
+    from kano_peripherals.pi_hat.driver.high_level import get_pihat_interface
+
```

Comment:
I appreciate that this probably speeds things up, but if it's not a functional change, it might be wiser to put it in a different PR that we merge after release. Or is there a subtle reason I'm missing?

Reply:
This was merely so that the tests don't fail because of missing imports - I can move it back and add the dependency to the container but it means that the tests will never pass on OSX